Geometry/DTGeometryBuilder/plugins/dd4hep/DTGeometryBuilder.cc

```diff
@@ -50,6 +51,8 @@ using namespace std;
 using namespace cms;
 
 void DTGeometryBuilder::buildGeometry(DDFilteredView& fview, DTGeometry& geom, const MuonGeometryConstants& num) const {
+  edm::LogVerbatim("DTGeometryBuilder") << "(0) DTGeometryBuilder - DD4Hep ";
+
```

Comment:
It looks like line 89 also needs conversion:
```
 new Plane(Surface::PositionType(tr[0], tr[1], tr[2]),
```
I believe these are positions in cm.